src/main.rs

```diff
@@ -103,8 +106,35 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let (shutdown_tx, mut shutdown_rx) = mpsc::channel::<()>(1);
 
+    let registry = SharedRegistry::global();
+    // Use the new non-global metrics registry when we upgrade to newer version of malachite
+    let _ = Metrics::register(registry);
+
+    let (rpc_block_tx, mut rpc_block_rx) = mpsc::channel::<Block>(100);
+    let (main_block_tx, mut main_block_rx) = mpsc::channel::<Block>(100);
+
+    let mut block_store = BlockStore::new();
+    while let Some(block) = main_block_rx.recv().await {
+        block_store.put_block(block)
+    }
```

Comment:
Why do we need this? Who's publishing at this point?

Reply:
Removed this. This code wouldn't work and not needed anymore because of the lock.